src/main/java/io/appium/java_client/remote/AndroidMobileOptions.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.appium.java_client.remote;
+
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.remote.CapabilityType;
+
+public class AndroidMobileOptions extends MobileOptions<AndroidMobileOptions> {
```

Comment:
I would say platform-specific option classes should go to android/ios packages

Reply:
Makes sense, I put them in `remote` because that's where `IOSMobileCapabilityType` and `AndroidMobileCapabilityType` were.

Reply:
3dd48dd54f1c7c6e54202b7e473a380e0e79424b